### ISSUE TITLE
Green tint with splatfacto-big and bilagrad 

### ISSUE THREAD

**Issue by gradeeterna** [nerfstudio-project/nerfstudio#3383]
Splatfacto-big with bilagrad enabled has a green tint, not present with regular splatfacto + bilagrad, or splatfacto-big with bilagrad disabled:

splatfacto-big + bilagrad enabled:
<img width="1276" alt="splatfactobig-bilagrad" src="https://github.com/user-attachments/assets/d7847811-6335-44a8-a318-1637596d5dce">

splatfacto + bilagrad enabled:
<img width="1277" alt="splatfacto-bilagrad" src="https://github.com/user-attachments/assets/c57c9b5f-c731-471f-a29a-2b2e24ff8c2d">

splatfacto-big + bilagrad disabled:
<img width="1275" alt="splatfactobig-nobilagrad" src="https://github.com/user-attachments/assets/101f2389-46b8-4803-9680-17dea65c343a">

splatfacto + bilagrad enabled + some arguments to match splatfacto-big looks fine also `--pipeline.model.densify-grad-thresh 0.0006 --pipeline.model.cull-alpha-thresh=0.005  --pipeline.model.continue-cull-post-densification=False`
<img width="1273" alt="splatfacto-diy-big-bilagrad" src="https://github.com/user-attachments/assets/91518fa5-7fdd-419b-929c-01db4638193f">

@SharkWipf commented on Discord that the only other difference between regular and big splatfacto is the learning rate:
![Screenshot_20240824-141552](https://github.com/user-attachments/assets/2e3c881c-0cde-4556-9d09-32d5b8219ebc)

I've only tested with this tricky dataset where the exposure is all over the place, but can test some others soon. 

Here is the ground truth image, and another over-exposed image from nearby:
<img width="845" alt="gt1" src="https://github.com/user-attachments/assets/eb7c168d-bd2c-4928-a355-002080d15357">
<img width="845" alt="gt2overexposed" src="https://github.com/user-attachments/assets/90787604-d0a2-43c9-a46d-fa5ab86e3b61">

**Comment by abrahamezzeddine** [comment]
Out of curiousity, how do I enable bilagird with splatfacto? I would like to try this myself.☺️ thanks!

**Comment by jb-ye** [comment]
@abrahamezzeddine `--pipeline.model.use_bilateral_grid True`

**Comment by xiemeilong** [comment]
Could you please tell me the difference in training FPS when using bilateral_grid and not using it? I've noticed a significant slowdown in my code after adding bilateral_grid and I'm wondering if this is expected behavior.

**Comment by KevinXu02** [comment]
Sorry for the late reply. I forgot to update the lr rate for splatfacto-big when tuning splatfacto. So it should be 2e-3 for splatfacto-big. 

The overhead for bilateral_grid should be scaled by the image resolution. There can be a 20% slowdown for 960p images, but may hurt more if use higer resolution images.
